io/src/test/java/org/commonjava/maven/ext/io/PomIOTest.java

```diff
@@ -48,6 +56,29 @@ public void setup()
         pomIO = new PomIO();
     }
 
+
+    @Test
+    public void testRoundTripPOMs()
+                    throws Exception
+    {
+        URL resource = PomIOTest.class.getResource( filename );
+        assertNotNull( resource );
+        File pom = new File( resource.getFile() );
+        assertTrue( pom.exists() );
+
+        File targetFile = folder.newFile( "target.xml" );
+        FileUtils.copyFile( pom, targetFile );
+
+        List<Project> projects = pomIO.parseProject( targetFile );
+        // We don't want this to be the execution root so that it doesn't add "Modified by" which breaks the comparison
+        FieldUtils.writeDeclaredField( projects.get( 0 ), "executionRoot", false, true);
+        HashSet<Project> changed = new HashSet<>(projects);
+        pomIO.rewritePOMs( changed );
+
+        assertTrue( FileUtils.contentEqualsIgnoreEOL( pom, targetFile, StandardCharsets.UTF_8.toString() ) );
```

Comment:
Charset could be `Charset.forName(  projects.get( 0 ).getModel().getModelEncoding() )`.

Reply:
I've moved everything to using StandardCharsets.UTF_8 as the model either has that or null (which breaks the tests)